packages/core/src/api/guild.ts

```diff
@@ -83,6 +83,7 @@ import {
 	type RESTPostAPITemplateCreateGuildJSONBody,
 	type RESTPutAPIGuildBanJSONBody,
 	type RESTPutAPIGuildTemplateSyncResult,
+	type RESTGetAPIGuildWebhooksResult,
```

Comment:
This import isn't sorted, it should be on line 41 (`RESTGetAPIGuildMembersSearchQuery` → `RESTGetAPIGuildWebhooksResult` → `RESTGetAPIGuildWelcomeScreenResult`).